config/sidekiq_cron_schedule.yml

```diff
@@ -25,4 +25,7 @@ check_no_induction_participants:
 enrol_school_cohorts:
   cron: "0 3 * * *"
   class: "EnrolSchoolCohortsJob"
+transition_statements_to_payable:
+  cron: "<%= Rails.env.production? ? '@daily' : '@hourly' %>"
```

Comment:
are you sure you have access to erb here? i only had a brief scan of the gem and don't see any erb handling in this part